src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -348,6 +352,11 @@ public bool VerifyData(Organization organization, IGlobalSettings globalSettings
             //     valid = organization.LimitCollectionCreationDeletion == LimitCollectionCreationDeletion;
             // }
 
+            if (valid && Version >= 14)
+            {
+                valid = organization.AllowAdminAccessToAllCollectionItems == AllowAdminAccessToAllCollectionItems;
+            }
```

Comment:
Should be commented our per the version 13 validation above.

This is because self-hosted orgs do not use this value unless the feature flag is turned on in the self-hosted instance, therefore we expect this check to fail in some circumstances. (e.g. cloud has the flag on, client updates the setting and then updates their license file, but the flag is off in self-hosted. The license value will be ignored, the org value won't match the license file, so this validation would otherwise fail.)

We can do this because validation of this property is not particularly important either (it's not billable).